istioctl/pkg/describe/describe.go

```diff
@@ -77,8 +78,16 @@ type myProtoValue struct {
 
 const (
 	k8sSuffix = ".svc." + constants.DefaultClusterLocalDomain
+
+	printLevel0 = 0
```

Comment:
I think this can be improved with a smarter formatter? May also check how kubectl handles this.

Reply:
great idea

Reply:
Not blocking the PR, I think this is not related to the fix and can be a follow-up.